src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -79,30 +80,30 @@ def set_top_tags(scope: Scope, project: Project) -> None:
 def try_path_munging(
     config: RepositoryProjectPathConfig,
     filepath: str,
-    mobile_frame: Any,
-    ctx: Any,
-) -> Any:
-    result = {}
-    mobile_frame["filename"] = filepath
+    mobile_frame: Mapping[str, Optional[str]],
+    ctx: Mapping[str, Optional[str]],
+) -> Mapping[str, str]:
+    result: Mapping[str, str] = {}
+    mobile_frame["filename"] = filepath  # type: ignore
```

Comment:
We will fix this ignore in future PRs.

Reply:
hmmm I think this is complaining because `Mapping` is meant to be readonly -- `MutableMapping` would work (though at that point I'd just use `dict` probably :shrug:)